app/routes/jokes/new.tsx

```diff
@@ -106,15 +68,15 @@ export default function NewJokeRoute() {
               type='text'
               defaultValue={actionData?.fields?.name}
               name='name'
-              aria-invalid={Boolean(actionData?.fieldErrors?.name) || undefined}
+              aria-invalid={fieldHasErrors(actionData, "name")}
```

Comment:
Nice! avoiding those pesky safe navigator operators is always easier on the eyes ;)